xclim/indices/run_length.py

```diff
@@ -786,7 +893,7 @@ def season(
     Returns
     -------
     xr.Dataset
-        "dim" is reduced to "season_bnds" with 2 elements : season start and season end, both indices of da[dim].
+        A Dataset with three variables : start, end and length of the season.
```

Comment:
I don't know why suggestion is not working.

You could follow the current convention in sdba

```python
    ds : xr.Dataset
        Dataset variables:
            start 
            end
            length
```